docs/server-config.md

```diff
@@ -41,3 +41,57 @@ TABPY_KEY_FILE = C:/path/to/key/file.key
 
 Note that only PEM-encoded x509 certificates are supported for the secure
 connection scenario.
+
+## Authentication
+
+TabPy supports basic access authentication (see
+[https://en.wikipedia.org/wiki/Basic_access_authentication](https://en.wikipedia.org/wiki/Basic_access_authentication)
+for more details).
+
+### Enabling Authentication
+
+To enable the feature specify `TABPY_PWD_FILE` parameter in TabPy
+configuration file:
+
+```sh
+TABPY_PWD_FILE = c:\path\to\password\file.txt
+```
+
+Password file is a text file containing usernames and hashed passwords
+per line separated by space. For username only ASCII characters
+supported.
+
+There is `utils/user_management.py` utility to operate with
+accounts in the password file. Run `utils/user_management.py -h` to
+see how to use it.
+
+Note for any of the changes for password file TabPy needs to be restarted.
```

Comment:
This would be better
After making any changes to the  password file, TabPy needs to be restarted.

Reply:
fixed